apps/rush/src/actions/RushCommandLineParser.ts

```diff
@@ -90,6 +92,15 @@ export default class RushCommandLineParser extends CommandLineParser {
     }
   }
 
+  private _ensureEnvironment(): void {
+    /* tslint:disable-next-line:no-string-literal */
+    let environmentPath: string = process.env['PATH'];
+    environmentPath = path.join(this.rushConfiguration.commonTempFolder, 'node_modules', '.bin') +
```

Comment:
I suppose that's okay.

---
In reply to: [119759143](https://github.com/Microsoft/web-build-tools/pull/218#discussion_r119759143) [](ancestors = 119759143,119754113,119753539,119751422)